api/interface.go

```diff
@@ -2,11 +2,14 @@ package api
 
 import (
 	"github.com/ambientsound/pms/songlist"
+
+	"github.com/gdamore/tcell/views"
 )
 
 type SonglistWidget interface {
 	AddSonglist(songlist.Songlist)
 	FallbackSonglist() songlist.Songlist
+	GetViewport() views.ViewPort
```

Comment:
In order for this to work you would write:

```
GetViewport() *views.ViewPort
```

Reply:
Oh wow, it's as simple as that. Thanks.